commands/tests/help.test.js

```diff
@@ -0,0 +1,50 @@
+const Discord = require('discord.js');
+const MockMessage = require('../../stub/MockMessage.js');
+const help = require('../help.js');
+
+const client = new Discord.Client();
+const message = new MockMessage();
+
+beforeAll(() => {
+	client.commands = new Discord.Collection();
+	client.aliases = new Discord.Collection();
+	['help', 'facebook','invite','coinflip'].forEach(name => 
+		client.commands.set(name, require(`../${name}`))
+	);
+});
+
+beforeEach(() => {
+	jest.clearAllMocks();
+});
+
+afterAll(() => {
+	client.destroy();
+});
+
+test('Sends default help message when no argument is provided', async () => {	
+	await help.execute(client, message, []);
+
+	const expectedResponse = 'I have sent you a private message with the command list.';
+	expect(message.author.send).toHaveBeenCalledTimes(1);
```

Comment:
Do we need to test if the embed DMed to the user is correct?

Reply:
Added this to the test